doc/source/replica_workflow.rst

```diff
@@ -0,0 +1,91 @@
+Typical replica workflow
+========================
+
+This section gives an overview of what happens within Rucio, for a typical replica workflow. Two workflows are described:
+When a replica is uploaded to Rucio via a client and when a replica is created by a
+site to site transfer due to the creation of a `replication rule`_.
+
+
+Replica pathes on storage
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rucio has two basic paradigms in deciding the path for a replica on a specific storage system. **Deterministic** and **Non-deterministic** pathes. If we assume
+a file whose data identifier is ``user.jdoe:test.file.1``, thus the scope is ``user.jdoe`` and the name is ``test.file.1``. In Rucio a deterministcally created path is a path
+which can be generated soley knowing the scope and name of a data identifier (Ignoring the static prefix of the storage endpoint). For a non-deterministic path
+additional information describing the file is necessary, such as meta-data, the dataset the file belongs to, etc.
+
+Rucio supports plugable algorithms for both deterministic and non-deterministic algorithms. This section explains a few of them.
+
+Deterministc algorithm based on hashes
```

Comment:
deterministic


(global search and replace ;-)

Reply:
Ups 🤣